frontend/awx/resources/inventories/hooks/useInventoriesHostsActions.tsx

```diff
@@ -7,25 +7,36 @@ import {
   PageActionType,
   usePageNavigate,
 } from '../../../../../framework';
-import { requestPatch } from '../../../../common/crud/Data';
 import { cannotDeleteResource, cannotEditResource } from '../../../../common/utils/RBAChelpers';
 import { awxAPI } from '../../../common/api/awx-utils';
 import { AwxRoute } from '../../../main/AwxRoutes';
 import { AwxHost } from '../../../interfaces/AwxHost';
 import { useDeleteHosts } from '../../hosts/hooks/useDeleteHosts';
+import { usePatchRequest } from '../../../../common/crud/usePatchRequest';
 
-export function useInventoriesHostsActions(onComplete: (hosts: AwxHost[]) => void) {
+export function useInventoriesHostsActions(
+  onDelete: (host: AwxHost[]) => void,
+  onToggle: ((host: AwxHost[]) => void) | ((host: AwxHost) => void),
```

Comment:
Hey @AlexSCorey the reason I put in two parameters here, on for onDelete and onToggle is that there are certain cases where the behaviour for onToggle is different from onDelete. On the inventory hosts lists section, we can pass the same callback for both onToggle and onDelete `view.unselectItemsAndRefresh`. On the inventory hosts details section, it's a little bit tricker because onDelete should redirect you to the lists page and onToggle should just update the hostState state variable. Maybe there is another way to do it but I'm not too sure